remoteappmanager/handlers/home_handler.py

```diff
@@ -265,8 +265,12 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         volumes = {}
 
         if mount_home:
-            home_path = os.path.expanduser('~'+user_name)
-            volumes[home_path] = {'bind': '/workspace', 'mode': 'rw'}
+            home_path = os.environ.get('HOME')
```

Comment:
even if you are running with virtual users, HOME will be defined (and it's the jupyterhub user). This will mount the jupyterhub home directory for all containers that have mount_home specified.